### DISCUSSION TITLE
Un tableau des scores

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2264] [Idées acceptées / déjà implémentées]
Salut! Je me permets de vous contacter afin de partager avec vous une suggestion qui, je pense, pourrait améliorer l'expérience de jeu et des combats 

Comme vous le savez, depuis que le temps de régen a été modifié depuis la dernière mise à jour la commande « /combat » est très populaire et utilisée fréquemment par les membres de notre communauté pour s’affronter et gagner des points Cependant, j'ai remarqué qu'il peut parfois être difficile de se souvenir contre qui nous avons combattu, combien de fois, et quels étaient les scores de ces affrontements, surtout lorsque l’on participe à de nombreux combats en peu de temps.

Pour remédier à cela, je souhaiterais proposer l'implémentation d'une nouvelle commande, par exemple « /tableau_combat », qui afficherait un résumé des combats Classés récents d'un joueur, incluant des informations telles que :

- Le nom de l'adversaire
- Le nombre total de combats contre cet adversaire
- Le bilan des victoires/défaites/nuls contre cet adversaire

Cette fonctionnalité permettrait à chacun de suivre ses performances, de se remémorer ses combats précédents, et d’apporter une dimension stratégique supplémentaire au jeu en permettant aux joueurs de préparer leurs prochains affrontements en fonction de leurs expériences passées.
Je ne sais pas si pour d'autres celle-ci serait vraiment utile ou si celle-ci est réalisable, mais je souhaitais proposer l'idée sur ce je vous souhaite une agréable soirée . (merci chatgpt pour avoir su expliquer mon idée )

**Comment by BastLast** [comment]
Hello honnêtement ça pourra être une fonctionnalité ajoutée un jour mais je vois mal ça arriver avant le v5 (aka plusieurs mois voir années) étant donné la complexité du développement nécessaire surtout pourbfakre une gui propre

**Reply by BastLast** [reply]
Mais je suis curieux de voir les votes sur cette suggestion

**Reply by Voisin1234** [reply]
Moi également mais j'avais l'idée l'autre jour elle a fait un flop quand j'en ai parler mais je souhaiter quand même la proposer.

**Comment by Voisin1234** [comment]
Un petit UP☺️

**Comment by Voisin1234** [comment]
Un grand up surtout depuis la v5

**Comment by BastLast** [comment]
#2615 similaire à ça
